lib/gui/app/pages/main/Flash.tsx

```diff
@@ -160,7 +163,11 @@ const formatSeconds = (totalSeconds: number) => {
 	return `${minutes}m${seconds}s`;
 };
 
-export const Flash = ({ shouldFlashStepBeDisabled, goToSuccess }: any) => {
+export const Flash = ({
+	shouldFlashStepBeDisabled,
+	goToSuccess,
+	source,
+}: any) => {
```

Comment:
Please avoid `any`

Reply:
this wasn't changed, but sure, pushing the fix now